docs/configuring-playbook-etherpad.md

```diff
@@ -30,12 +32,14 @@ To enable Etherpad, add the following configuration to your `inventory/host_vars
 
 ```yaml
 etherpad_enabled: true
-
-# Uncomment and adjust this part if you'd like to enable the admin web UI
-# etherpad_admin_username: YOUR_USERNAME_HERE
-# etherpad_admin_password: YOUR_PASSWORD_HERE
```

Comment:
It seems like these are getting lost and they could be quite important for protecting your instance from anonymous users using it.

Reply:
Hopefully https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/4066/commits/91d991cac6d497f0383a31a2c0e760953716b427 should address this :slightly_smiling_face: